modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -152,6 +154,10 @@ export default {
       type: String,
       default: null
     },
+    nonForeignWidgetHovered: {
```

Comment:
this prop doesn't get cleaned up (as far as i can tell) so it can't be used to help the UI know when we've stopped hovering foreign widgets